src/Utilities/StdConstexprHelpers.hpp

```diff
@@ -0,0 +1,116 @@
+// Distributed under the MIT License.
```

Comment:
We had discussed putting these in `Utilities/Algorithm.hpp`, `Utilities/Numeric.hpp`, etc. depending on which STL header they were in. Could you do that?

Reply:
Sure. After thinking some more, I thought it made more sense to have these in one helper file, since they are all constexpr rewrites of STL stuff. But I don’t feel strongly about it,  just wanted to propose it. I’ll change it back to what we had discussed